spec/request/organizations_spec.rb

```diff
@@ -1467,6 +1468,31 @@ module VCAP::CloudController
           end
         end
 
+        it_behaves_like 'list query endpoint' do
```

Comment:
Maybe it would make more sense to add the new tests to `spec/request/users_spec.rb` and exclude the parameter from `spec/request/organizations_spec.rb` and `spec/request/spaces_spec.rb`.

Reply:
I'm thinking that maybe it will be more valuable to have tests fro the other two. Therefore, we'll have to test the new queryParam on the missing endpoints (users and space users). 
Althought, the main functionality is the partial search on the organization users list.